src/chains.ts

```diff
@@ -351,6 +351,231 @@ export const supportedChains: Array<Chain> = [
     },
   },
 
+  // 2 - Expanse Network
+  {
+    key: ChainKey.EXP,
+    name: 'Expanse Network',
+    coin: CoinKey.EXP,
+    id: 2,
+    visible: false,
+    tokenlistUrl: '', // TODO: find token list url
+
+    metamask: {
+      chainId: prefixChainId(2),
+      blockExplorerUrls: ['https://expanse.tech'],
```

Comment:
Please check that the `blockExplorerUrls` always end with a `/` this makes building urls with them easier.

Reply:
okay, got it